pkg/bindings/test/images_test.go

```diff
@@ -403,7 +403,6 @@ var _ = Describe("Podman images", func() {
 		Expect(output).To(ContainSubstring("Copying blob "))
 		Expect(output).To(ContainSubstring("Copying config "))
 		Expect(output).To(ContainSubstring("Writing manifest to image destination"))
-		Expect(output).To(ContainSubstring("Storing signatures"))
```

Comment:
I don't know the reasoning behind this change, nor how important it is to preserve it for the future, but would it make sense to change at least some of these tests to `NotTo(Contain()` (<---- deliberate shortcut, not exact code) instead of removing the tests?

Reply:
I can do that if checking that makes sense, @mtrmac WDYT?

Reply:
Makes sense, that we should only see that line, when we are "Storing signatures"

Reply:
> Is there a easy way for us in podman to make a test that actually stores signatures?

For pushes, a push with `--sign*`; e.g. https://github.com/containers/podman/blob/3a013b6256a407ce13cc7131b4bad8230ecc383d/test/e2e/push_test.go#L140 .

For pulls, a pull of a signed image (with `registries.d` set up to read signatures). One instance where that is guaranteed to happen is also that test. (Alternatively, we could pull UBI or something like that, relying on the registry to be up.)

Reply:
Note In the interest of time I left this comment open, I don't like checking for something is not unless you have another test checking that it is. Given the release dance I like to get this PR in, if anyone thinks testing this is important I can follow-up with another PR.